docs/user-guides/filtering-configuration.md

```
@@ -153,7 +153,7 @@ func main() {
 
 	// connect to the Vald cluster
 	ctx := context.Background()
-	conn, err := grpc.DialContext(ctx, grpcServerAddr, grpc.WithInsecure())
+	conn, err := grpc.NewClient(grpcServerAddr, grpc.WithInsecure())
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)